src/applications/disability-benefits/bdd/config/form.js

```diff
@@ -104,12 +104,12 @@ const formConfig = {
     environment.API_URL
   }/v0/disability_compensation_form/submit_all_claim`,
   submit: submitFormFor('disability-526EZ'),
-  trackingPrefix: 'disability-526EZ-bdd-',
```

Comment:
@ndsprinkle - correct me if I'm wrong, but I believe this reverts work you recently merged in here - https://github.com/department-of-veterans-affairs/vets-website/pull/12810